ecs_pod_creator.go

```diff
@@ -236,6 +326,253 @@ func (e *EnvironmentVariable) SetSecretOptions(opts SecretOptions) *EnvironmentV
 	return e
 }
 
+// Validate checks that the environment variable name is given and that either
+// the raw environment variable value or the referenced secret is given.
+func (e *EnvironmentVariable) Validate() error {
+	catcher := grip.NewBasicCatcher()
+	catcher.NewWhen(e.Name == nil, "must specify a name")
```

Comment:
Should we also check that the name is not empty when it is specified?

Reply:
Yes, fixed.